action.yml

```diff
@@ -50,16 +62,28 @@ runs:
           EXTRA_ARGS="${EXTRA_ARGS} --outputColumns ${output_column}"
         done
         
+        for output_column in ${REUSABLE_OUTPUT_COLUMNS[@]}; do
+          EXTRA_ARGS="${EXTRA_ARGS} --reusableOutputColumns ${output_column}"
+        done
+
+        for secret_column in ${SECRETS_COLUMNS[@]}; do
+          EXTRA_ARGS="${EXTRA_ARGS} --secretsColumns ${secret_column}"
+        done
+
+        if [[ "$REUSABLE" == “true” ]]; then
```

Comment:
There's some unicode quotes here, fixed in https://github.com/andreas-aman/auto-doc/pull/2/files. With this change made, this PR seems to work great!

I used the following horrific hack to test it:
```yaml
      - name: Download auto-doc
        id: autodoc
        env:
          GH_TOKEN: ${{ github.token }}
        run: |
          mkdir -p /tmp/auto-doc /tmp/gopath
          oldcd=$(pwd)
          cd /tmp/auto-doc
          git clone https://github.com/tj-actions/auto-doc.git .
          gh pr checkout 405
          go install
          cd "$oldcd"
          rm -rf /tmp/auto-doc
          BIN_PATH="${GOPATH:-$HOME/go}/bin/auto-doc"
          "$BIN_PATH" -h
          echo "BIN_PATH=$BIN_PATH" >> "$GITHUB_OUTPUT"

      - name: Run auto-doc
        uses: Makeshift/auto-doc@reusable # Should be replaced when this PR is merged: https://github.com/tj-actions/auto-doc/pull/405
        with:
          bin_path: ${{ steps.autodoc.outputs.BIN_PATH }}
          filename: '.github/workflows/workflow.yml'
          reusable: true
```

Reply:
I wonder from where these unicode quotes came? Nice catch!